scripts/src/main/resources/scripts/command/docker

```diff
@@ -67,13 +69,12 @@ function doSetup() {
           doSetVpnKitStartupInstaller
           doEcho "Installing Rancher Desktop for Windows..."
           # ensures installation of vpnkit if user directly restarts computer for enabling wsl
-          doRunCommand "${rancher_install_dir}/rancher-*-windows.exe" 
+          doRunCommand "${rancher_install_dir}/rancher-*-windows.exe"
           #check if vpnkit is already installed
           doEnsureVpnKitInstallation
         elif doIsMacOs
         then
-          # shellcheck disable=SC2016
-          doRunCommand 'cp -a "${rancher_install_dir}"/*.app /Applications/'
+          doRunCommand "cp -a ${rancher_install_dir}/*.app /Applications/"
```

Comment:
IMHO you introduced a bug here in case the path (`${rancher_install_dir}`) would contain a whitespace.
Can you put single quotes around it due to #100

Reply:
As constructive review, I simply added the quotes directly so we can already merge your great work.